prac_06/languages.py

```diff
@@ -0,0 +1,13 @@
+from prac_06.programming_language import ProgrammingLanguage
```

Comment:
I included a docstring for this file noting it was client code for a program to use the ProgrammingLanguage class